Civi/Share/Message.php

```diff
@@ -103,11 +106,11 @@ public function send($local_node_id = null)
       foreach ($peerings as $peered_node) {
```

Comment:
Let's rename `$peered_node` to `$peering`, as it is not holding the node itself.

Reply:
you, we could do that